scripts/config/sonar-scanner.properties

```diff
@@ -4,6 +4,7 @@ sonar.host.url=https://sonarcloud.io
 sonar.qualitygate.wait=true
 sonar.sourceEncoding=UTF-8
 sonar.sources=.
+sonar.exclusions=docs/**,scripts/**
```

Comment:
`docs/**` is fine, but I don't think we want to exclude `scripts/**`.  There's quite a lot of shell and make in there that we do want covered.

I've also removed the python `hello_world` example so the original motivating failures for this PR should (I think) have gone away.